docs/source/cognite.rst

```diff
@@ -13,12 +13,15 @@ You can also pass your API key directly to the CogniteClient.
 .. code:: python
 
     >>> from cognite.client import CogniteClient
-    >>> c = CogniteClient(api_key="your-key")
+    >>> c = CogniteClient(api_key="<your-api-key>", client_name="your-client-name")
 
 Instantiate a new client
 ------------------------
 Use this code to instantiate a client and get your login status. CDF returns an object with
-attributes that describe which project and service account your API key belongs to.
+attributes that describe which project and service account your API key belongs to. The client_name
+is used to give a unique identifier to the client. You can choose any string for the client_name. The client_name
+can also be provided through the :code: `COGNITE_CLIENT_NAME` environment variable.
```

Comment:
Replace "The client_name can also be provided through..." with "You can also provide the client_name through..."